bigchaindb/common/utils.py

```diff
@@ -97,6 +97,30 @@ def validate_all_keys(obj_name, obj, validation_fun):
     return
 
 
+def validate_all_value_for_key(obj, key, validation_fun):
+    """Validates value for all (nested) occurences of `key` in `obj`
+       using `validation_fun`
+
+        Args:
+            obj (dict): dictonary object.
+            key (str): key whose value is to be validated.
+            validation_fun (function): function used to validate the value
+            of `key`.
+
+        Returns:
+            None: indicates validation successfull
```

Comment:
This function doesn't provide to the caller any useful return value, what do you think about removing the `Returns` part? As an alternative idea, you can return `obj` on success.

Reply:
We can remove the `Returns` as its exactly correct. Furthermore, I think we could return `True` instead of `obj` because intutively a validation function should return `True` or `False`?